packages/yourloops/components/dialogs/join-team/privacy-policy-confirm.tsx

```diff
@@ -53,7 +53,6 @@ export const PrivacyPolicyConfirm = (props: ConfirmTeamProps): JSX.Element => {
   const { onCompleteStep, team, onClickCancel, inProgress } = props
   const [policyChecked, setPolicyChecked] = useState<boolean>(false)
   const theme = useTheme()
-
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:**  You really don't like to aerate your code, do you?‏